extensions/VisualEditor/modules/ve-mw/init/targets/ve.init.mw.ViewPageTarget.js

```diff
@@ -587,12 +607,30 @@ ve.init.mw.ViewPageTarget.prototype.onSaveErrorCaptcha = function () {
 			'callback': function ( response ) {
 				this.captchaResponse = response;
 			}.bind( this )
-		}
-	);
-	this.saveDialog.$frame.addClass( 'oo-ui-window-frame-captcha' );
-	this.saveDialog.popPending();
+		});
+};
 
-	this.events.trackSaveError( 'captcha' );
+/**
+ * If reCaptcha failed to load (eg, the user is in China and google is blocked),
+ * load Fancy Captcha and render it instead.
+ *
+ * @method
+ */
+ve.init.mw.ViewPageTarget.prototype.loadAndRenderFancyCaptcha = function () {
+	$.when(
+		$.getResources([
+			$.getSassCommonURL('extensions/wikia/Captcha/styles/FancyCaptcha.scss')
```

Comment:
This won't currently get applied since the save dialogue is in an iframe, however @inez reports that the iframes are going away in the next 3-4 weeks at which point these styles will start working. Note the FancyCaptcha.scss are getting applied everywhere outside of VE.